### ISSUE TITLE
Error in debian package

### ISSUE THREAD

**Issue by nunodonato** [MisterDA/love-release#15]
Hi! I just tried the new version with the icon support for debian packages.
Got it to work only after realizing that the icon folder needs to be inside the project folder, otherwise it didn't.

Anyway, the reason I'm writing another issue is because something else seems to be broken. After I installed the .deb with the package manager and launched the game from the menu, I got the Love Baby Inspector "no game!". Upon checking the bin file, found this:

> love /tmp/tmp.Vo5j0c5LpZ/usr/share/games/autumn/autumn-1.love

I think the /tmp part should not be there :)
